pandas/tests/frame/test_apply.py

```diff
@@ -121,13 +121,10 @@ def test_apply_standard_nonunique(self):
         assert_series_equal(rs, xp)
 
     @pytest.mark.parametrize('arg', ['sum', 'mean', 'min', 'max', 'std'])
-    def test_with_string_args(self, arg):
-        result = self.frame.apply(arg)
-        expected = getattr(self.frame, arg)()
-        tm.assert_series_equal(result, expected)
-
-        result = self.frame.apply(arg, axis=1)
-        expected = getattr(self.frame, arg)(axis=1)
+    @pytest.mark.parametrize('kwds', [{}, {'axis': 1}, {'numeric_only': True}])
```

Comment:
can you add a test for just std that adds both a positional arg (e.g. ddof) and a kwarg (numeric_only) for confirmation here (and other combinations if possible)